torch_xla/csrc/runtime/stablehlo_helper.cc

```diff
@@ -192,4 +192,16 @@ GetHloDtypeToStablehloDtypeMap() {
   return m_;
 }
 
+const std::unordered_map<std::string, xla::PrimitiveType>&
+GetTorchIntDtypeToHloDtypeMap() {
+  static const std::unordered_map<std::string, xla::PrimitiveType> m_{
```

Comment:
We could implement this as a switch / series of if stmts to avoid a static dictionary, it's a short enough list. Not sure of PT/XLAs stance on static data.

Reply:
I agree about less static data floating around. Replaced the dictionary with  conditionals. 
@lsy323 If the change LG to you then I can merge once the CI is green.